lib/http.js

```diff
@@ -27,6 +28,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       instance: '/vc-issuer/instances/:instanceId',
       instanceIssuer: '/vc-issuer/instances/:instanceId/issuer',
       automatedIssue: '/vc-issuer/issue',
+      issueCredential: '/issuers/:issuerId/credentials',
```

Comment:
Should this line up with the other endpoints?
```suggestion
      issueCredential: '/vc-issuer/instances/:instanceId/credentials',
```

Reply:
it should be the way it is `/issuers/:issuerId/credentials`.  This was decided on this morning in a design meeting.